### ISSUE TITLE
Failing due to go.jona.me/crowd

### ISSUE THREAD

**Issue by rskumar** [jda/go-crowd#3]
Simple program - 
```
package main
import "log"
import "github.com/jda/go-crowd"

func main() {
	c, err := crowd.New("demo", "demo", "http://localhost:8095/crowd")
	if err != nil {
		log.Fatal(err)
	}
	_, err := c.Authenticate("user_a", "pass_a")
	if err != nil {
		log.Fatal(err)
	}
}
```
Throws error 
```
../sandbox/atlassian_crowd/main.go:5:2: code in directory /home/xyz/goproj/src/github.com/jda/go-crowd expects import "go.jona.me/crowd"
```

**Comment by jda** [comment]
Thanks for reporting this. It should be fixed in commit 415c27e.

**Comment by rskumar** [comment]
Thanks, it works now.
